src/Blueprint/Blueprint/Common/Utils.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+
+namespace Microsoft.Azure.PowerShell.Cmdlets.Blueprint.Common
+{
+    public class Utils
+    {
+        /// <summary>
+        /// Get definition location. Either a subcscription or a management group.
+        /// </summary>
+        /// <param name="scope"></param>
+        /// <returns></returns>
+        public static string GetDefinitionLocationId(string scope)
+        {
+            if (string.IsNullOrEmpty(scope))
+                return null;
+
+            string[] tokens = scope.Split(new[] {'/'}, StringSplitOptions.RemoveEmptyEntries);
+            string locationId = null;
+
+            if (tokens != null && tokens.Length == 2)
+            {
+                locationId = string.Equals(tokens[0], "subscriptions", StringComparison.InvariantCultureIgnoreCase) ? tokens[1] : null;      
```

Comment:
>InvariantCultureIgnoreCase [](start = 88, length = 26)

We're doing OrdinalIgnoreCase in the backend. Same comment for the comparison below as well.